daemon/cmd/daemon_main.go

```diff
@@ -1373,6 +1376,11 @@ func runDaemon() {
 	metricsErrs := initMetrics()
 
 	d.startAgentHealthHTTPService(fmt.Sprintf("localhost:%d", option.Config.AgentHealthPort))
+	if option.Config.KubeProxyReplacementHealthzBindAddr != "" {
+		if option.Config.KubeProxyReplacement == option.KubeProxyReplacementDisabled {
```

Comment:
Should this be `!=`?

Reply:
The answer is yes. If '==' then we don't need to add a bind address if not '!=' we need to add a bind address.
Am I missing something.